cylc/flow/scripts/clean.py

```diff
@@ -18,7 +18,10 @@
 
 """cylc clean [OPTIONS] ARGS
 
-Remove a stopped workflow from the local scheduler filesystem and remote hosts.
+Delete a stopped workflow.
+
+Remove workflow files from the local scheduler filesystem and any remote hosts
```

Comment:
Not sure on this one, maybe filesystem is one word, just flagging in case it needs a space.

```suggestion
Remove workflow files from the local scheduler file system and any remote hosts
```